tests/UI/specs/JSTracker_spec.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Piwik - free/libre analytics platform
+ *
+ * Bar graph screenshot tests.
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+describe("JSTracker", function () {
+    this.fixture = 'Piwik\\Tests\\Fixtures\\JSTrackingUIFixture';
+
+    var testWebsiteUrl = 'tests/resources/overlay-test-site-real/index.html';
+    var generalParams = 'idSite=1&period=day&date=today',
+        widgetizeParams = "module=Widgetize&action=iframe",
+        visitorLogUrl = "?" + widgetizeParams + "&" + generalParams + "&moduleToWidgetize=Live&actionToWidgetize=getVisitorLog";
+
+    it("run correctly on a website and correctly track visits in the visitor log", async function () {
+        await page.goto(testWebsiteUrl);
+
+        // view another page
+        await page.evaluate(() => $('a:contains(Page 3)')[0].click());
+        await page.waitForNetworkIdle();
+        await page.waitFor(500);
+        await page.waitForNetworkIdle();
+
+        // visit visitor log for correct date
+        await page.goto(visitorLogUrl);
+
+        expect(await page.screenshot({ fullPage: true })).to.matchImage('visitor_log');
```

Comment:
the expected image for that test is missing

Reply:
Seems something doesn't work here. Last processed screenshot shows en empty visitor log:

https://builds-artifacts.matomo.org/matomo-org/matomo/new-js-tracking-test/38545/processed/JSTracker_visitor_log.png

Reply:
Yeah, it works when it runs by itself, but not w/ other tests. Will be fun to debug.